### PR TITLE
fix broken KUBEVIRTCI_LOCAL_TESTING.md link

### PR DESCRIPTION
**Pull request by RamLavi**
file was renamed in previous PR #576
fixing link.

Signed-off-by: Ram Lavi <ralavi@redhat.com>

### CODE REVIEW

**Review by oshoval — APPROVED** [review]
Thanks

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ See [K8S.md](K8S.md)
 
 ## Testing kubevirt with locally provisioned cluster
 
-See [KUBEVIRT_LOCAL_TESTING.md](KUBEVIRT_LOCAL_TESTING.md)
+See [KUBEVIRTCI_LOCAL_TESTING.md](KUBEVIRTCI_LOCAL_TESTING.md)
```
Comment (oshoval) [inline README.md:11]: maybe worth to change also line 9

**Review by oshoval — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RamLavi) [reply README.md:11]: [fixed](https://github.com/kubevirt/kubevirtci/compare/2799113dc1c1b4c0c7a9babb44129833e0a16745..6a8e6e930340757e6ca4003f924f5aef6161a9ce)

**Review by oshoval — APPROVED** [review]

**Review by phoracek — COMMENTED** [review]
/approve

**Comment by oshoval** [comment]
please rebase

**Comment by oshoval** [comment]
@RamLavi 
please consider to reopen as we still need it (at lease last time i checked)

**Pull request closed without merging**